packages/tocco-ui/src/IconTocco/StyledIconTocco.js

```diff
@@ -0,0 +1,132 @@
+import styled, {keyframes} from 'styled-components'
```

Comment:
the main problem i see with this new loading image is that sometimes we use the loading image multiple times in a row on the same position (loading of entity browser and then loading of list , which is a part of entity browser).
When it gets shown the logo looks like one loading mask with a laggy animation. But what really is shown are two independent loading masks and the second one starts the animation from the beginning... But i think there is nothing we can do about that?